README.md

```diff
@@ -2,36 +2,40 @@
 Clone or fork this repo
 run `npm i`
 
+### Three database options:
+- `dev`: personal testing database, the default option for development. Create your own credentials to access
+- `staging`: group testing
+- `production`: group production 
 
 ### Needed in .env file:
 ```
-NODE_ENV="production"
+NODE_ENV= "production" || "staging" || "dev"
+AWS_SECRET=
+AWS_ID=
+
 # used if NODE_ENV === "production"
-FIREBASE_API_KEY=
-FIREBASE_AUTH_DOMAIN=
-FIREBASE_DB_URL=
-FIREBASE_ID=
-STORAGE_BUCKET=
-MESSAGING_SENDER_ID=
 FIREBASE_TOKEN=
 FIREBASE_EMAIL=
 
-# used if NODE_ENV !== "production"
-TESTING_FIREBASE_API_KEY=
-TESTING_FIREBASE_AUTH_DOMAIN=
-TESTING_FIREBASE_DB_URL=
-TESTING_FIREBASE_ID=
-TESTING_STORAGE_BUCKET=
-TESTING_MESSAGING_SENDER_ID=
-TESTING_FIREBASE_TOKEN=
-TESTING_FIREBASE_EMAIL=
+# used if NODE_ENV === "staging"
+STAGING_FIREBASE_TOKEN= 
+STAGING_FIREBASE_EMAIL=
 
-AWS_SECRET=
-AWS_ID=
+# used if NODE_ENV === "dev"
+DEV_FIREBASE_TOKEN=
+DEV_FIREBASE_EMAIL=
+DEV_FIREBASE_ID=
+DEV_FIREBASE_DB_URL=
 ```
+> To access `AWS`, `production`, or `staging`, please contact the development team for the necessary credentials 
+
 # To process a dataset:
 
 ## Setup
+### Set up a dev database
+- set NODE_ENV="dev"
+- add your required secret tokens to .env file 
```

Comment:
We should have at least a link to a tutorial on setting up a firebase database. And some information on where to find the particular variable you need and what they're called in firebase

Reply:
Imagine a new developer is going to start working on this: we want them to be able to set up the repo and start working just by reading the README and not having to ask you or I how to do it